Checks

```diff
@@ -6,3 +7,63 @@ check :no_api_keys do
     end
   end
 end
+
+check :integration_format do
+  puts ""
+  overviewimage = Array.new
+  headings = Array.new
+  self.output_filenames.each do |fn|
+    missingfiles =""
+    if fn.include? "output/integration"
+      doc = File.open(fn) { |f| Nokogiri::HTML(f)}
+      integrationlocation = fn.sub('output', 'content').sub('/index.html', '')
+      pagetitle = doc.xpath("//h1[@id='pagetitle']/text()")
+      newhlevel = doc.xpath("//div[@id='pagecontent']")
+
+      if newhlevel.length > 0 
+        #############
+        # New hlevel
+        #############
+
+        # Does the document have the right headings?
+        headings = doc.xpath("//div[@id='pagecontent']/h1").collect {|node| node.text.strip}
+        if headings.include? 'Overview'
+          if (headings & ['Configuration', 'Installation']).empty?
+            missingfiles += "\n  - Missing Config or Installation section"
+          else
+            overviewimage = doc.xpath("//img[preceding::h1[contains(text(), 'Overview')] and following::h1[(contains(text(), 'Configuration')) or (contains(text(), 'Installation'))]]/@src")  
```

Comment:
@technovangelist I was still getting the "missing overview image" on my [Route 53 doc](https://github.com/DataDog/documentation/pull/839/files#diff-01762e5e6b207681b1e38cce59ae38b0R13) and I think there are 2 issues:

1. Images are getting wrapped in P tags and it seems that the "preceding" selector is nearly the same as "preceding-sibling" and [doesn't include ancestors](https://www.w3.org/TR/xpath/#axes)... so when an img is on it's own, it has no preceding elements.
2. contains() also didn't seem to be working for me... and I'm not sure why. But unless someone is changing the id's, they should just be "overview", "configuration" and "installation".

In any case, this xpath seemed to work better for me:

```
overviewimage = doc.xpath("//p[preceding::h1[@id='overview'] and following::h1[(@id='configuration') or (@id='installation')]]/img/@src")
```